hex_dump/views.py

```diff
@@ -226,15 +227,34 @@ def extract_pc_hex(filename):
     return pc_hex_dict
 
 
-def simulate_bash_script(file_name , mtype, ctype, ftype, dtype):
-
-    assemble_cmd = ["riscv32-unknown-elf-gcc",f"-march=rv32i{mtype}{ctype}{ftype}{dtype}", "-mabi=ilp32", "-T", LINKER_SCRIPT, "-static", "-mcmodel=medany", "-fvisibility=hidden", "-nostdlib", "-nostartfiles", "-g", "-o", TMP_ELF, file_name]
+def simulate_bash_script(file_name , mtype, ctype, ftype, dtype , rvtype):
+    extensions = mtype + ctype + ftype + dtype
+    assembletype = 'riscv32'
+    abi = 'ilp32'
```

Comment:
Move the `assembletype` and `abi` variable definitions inside the `if rvtype == "rv32"` clause.